crates/agglayer-node/src/rate_limiting/local/mod.rs

```diff
@@ -0,0 +1,63 @@
+//! Local rate limiter, i.e. one constraining a single network.
+
+use agglayer_config::rate_limiting::TimeRateLimit;
+use tokio::time::Instant;
+
+// TODO: make private if possible
+mod interface;
+pub mod trivial;
+pub mod wall_clock;
+
+pub use interface::RateLimiter;
+
+pub struct LocalRateLimiter {
+    /// Rate limiter for `sendTx` settlement.
+    send_tx: trivial::RateLimiter<wall_clock::RateLimiter>,
+}
+
+impl LocalRateLimiter {
+    pub fn from_config(send_tx_config: &TimeRateLimit) -> Self {
```

Comment:
If we have more RPC added to this struct, it means that the arguments of this method will change right?
Could we use the base `config` and build the proper argument inside the function?
If we need to, we can have this method returning a `Result`.